src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke.command;
+
+import duke.exception.DukeException;
+
+import duke.taskTypes.Task;
+
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+public class DoneCommand extends Command {
+
+    private final String chosenList;
+
+    /**
+     * Basic Constructor
+     *
+     * @param storage Storage object to save
+     * @param taskList Tasklist to add task to
+     * @param ui Ui to display msg
+     * @param chosenList contains the string that describes which task to be declared done
+     */
+    public DoneCommand(Storage storage, TaskList taskList, Ui ui, String chosenList) {
+        super(storage, taskList, ui);
+        this.chosenList = chosenList;
+    }
+
+    /**
+     * Executes a set of instructions
+     *
+     * @return boolean To relay whether to continue the project
```

Comment:
I'm a little confused about what "continue the project" means here, could you clarify?

Reply:
The returned boolean would let duke decide whether to continue the while loop in Duke.java .and continue waiting for user input. Thinking back, this boolean could be set as a variable in Task and checking could be done by a method call.